asm/opcode_test.go

```diff
@@ -0,0 +1,49 @@
+package asm
+
+import (
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func TestGetSetJumpOp(t *testing.T) {
+	test := func(class Class, op JumpOp, valid bool) {
+		opcode := OpCode(class).SetJumpOp(op)
+
+		if valid {
+			qt.Assert(t, opcode, qt.Not(qt.Equals), InvalidOpCode)
+			qt.Assert(t, opcode.JumpOp(), qt.Equals, op)
+		} else {
+			qt.Assert(t, opcode, qt.Equals, InvalidOpCode)
+			qt.Assert(t, opcode.JumpOp(), qt.Equals, InvalidJumpOp)
+		}
+	}
+
+	// Exit, call and JA aren't allowed with Jump32
+	test(Jump32Class, Exit, false)
```

Comment:
All of these tests are going to assert on the same line, can you turn this into a table driven test instead?